appveyor.yml

```diff
@@ -126,8 +126,7 @@ before_deploy:
   - "SET PATH=%ORIGPATH%"
   - "%CMD_IN_ENV% %WP_INSTDIR%/scripts/env.bat"
   - "%CMD_IN_ENV% python.exe setup.py install"
-  - "%CMD_IN_ENV% easy_install --upgrade pip"
-  - "%CMD_IN_ENV% pip install --upgrade configobj traitsui"
+  - "%CMD_IN_ENV% pip install --upgrade configobj traitsui start_jupyter_cm"
```

Comment:
For me, `pip install start_jupyter_cm` creates two `.exe` files in my script folder with absolute paths to the python executable embedded in the shebang. This means that the python path as it is on the AppVeyor server will be embedded in the bundle. Consider patching using winpython methods?